src/com/hms/flexyosisoftconnector/OSIsoftConfig.java

```diff
@@ -357,6 +361,27 @@ public static void setFlexyName() {
     flexyName = res;
   }
 
+  /** Read the Flexy serial number and store it. */
+  public static void setFlexySerial() {

```

Comment:
I think this should probably be #readFlexySerial(). setFlexySerial() makes it seem like the serial number is mutable.